docs/operator-manual/web_based_terminal.md

```diff
@@ -0,0 +1,45 @@
+# Web-based Terminal
+
+![Argo CD Terminal](../assets/terminal.png)
```

Comment:
We already have 13 mb of images in `assets` folder. It is growing quickly and what is even worse stays in git history forever.

Instead of storing images in argocd repo let's store them in e.g. https://github.com/argoproj/argo-cd.wiki.git ? Or dedicated may be we can create dedicated https://github.com/argoproj/assets.git repo? 

@crenshaw-dev , @jannfis WDYT?

Reply:
Agreed that this will continue to grow. readthedocs supports git submodules, so we could have a separate assets repo and just link it that way.

Nonetheless, I compressed the image and force-pushed. Now it's only ~200kb.

Reply:
I'm cool with keeping them in a 2nd repo and then serving either from a submodule or directly. @alexmt can you create the assets repo?

Reply:
Sounds good! I want to discuss one more time assets repo toping during contributors meeting.

Thank you for reducing image size @notfromstatefarm ! Since it is 200kb now I'm fine to merge PR as is and move images to assets repo later.